modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -154,9 +156,47 @@ public Page<ObjectEntry> getCurrentObjectEntriesObjectRelationshipNamePage(
 			_objectEntryManagerServicesTracker.getObjectEntryManager(
 				_objectDefinition.getStorageType());
 
-		return objectEntryManager.getObjectEntryRelatedObjectEntries(
-			_getDTOConverterContext(currentObjectEntryId), currentObjectEntryId,
-			_objectDefinition, objectRelationshipName, pagination);
+		Page<ObjectEntry> objectEntryPage =
+			objectEntryManager.getObjectEntryRelatedObjectEntries(
+				_getDTOConverterContext(currentObjectEntryId),
+				currentObjectEntryId, _objectDefinition, objectRelationshipName,
+				pagination);
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipService.getObjectRelationship(
+				_objectDefinition.getObjectDefinitionId(),
+				objectRelationshipName);
+
+		ObjectDefinition objectDefinition2 =
+			_objectDefinitionLocalService.getObjectDefinition(
+				objectRelationship.getObjectDefinitionId2());
+
+		for (ObjectEntry item : objectEntryPage.getItems()) {
+			Map<String, Map<String, String>> itemActions = item.getActions();
+
+			for (Map.Entry<String, Map<String, String>> entry :
+					itemActions.entrySet()) {
+
+				Map<String, String> stringStringMap = entry.getValue();
+
+				String href = stringStringMap.get("href");
+
+				stringStringMap.put(
+					"href",
+					StringUtil.replace(
+						href,
+						StringUtil.lowerCaseFirstLetter(
+							_objectDefinition.getPluralLabel(
+								contextAcceptLanguage.getPreferredLocale())),
+						StringUtil.lowerCaseFirstLetter(
+							objectDefinition2.getPluralLabel(
+								contextAcceptLanguage.getPreferredLocale()))));
+			}
+
+			item.setActions(itemActions);
```

Comment:
I am not sure if this is necessary, maybe you are already updating the elements by reference

Reply:
True, good catch!